apps/voting/contracts/Voting.sol

```diff
@@ -205,15 +210,6 @@ contract Voting is IForwarder, AragonApp {
         vote.minAcceptQuorumPct = minAcceptQuorumPct;
 
         StartVote(voteId);
-
-        if (canVote(voteId, msg.sender)) {
-            _vote(
-                voteId,
-                true,
-                msg.sender,
-                true
-            );
-        }
```

Comment:
To avoid making someone do two calls to vote for a vote they create, maybe we could use a bool param to indicate whether they'd like to auto vote or not?

To avoid breaking changes, we could overload `newVote()` with a version that automatically passes true for auto voting.